### ISSUE TITLE
Threat feeds should support python3

### ISSUE THREAD

**Issue by VectraStuart** [vectranetworks/vectra_api_tools#14]
### Environment
Operating system:  Linux
Python version:  3
VAT version:  latest
Vectra version:  4.11

### Expected behavior

Threat feeds works in python3

### Actual behavior

Threat feeds does not work in python3, the print statements need parentheses

### Steps to reproduce

Run the threat feed in python3.

**Comment by HoneyBaked** [comment]
I agree, the threat feeds sample script needs to be updated to support python 3.
